tests/tt_eager/python_api_testing/non_working_unit_tests/grayskull/test_eltwise_addcdiv_stuck_1.py

```diff
@@ -49,7 +48,7 @@ def run_addcdiv(input_shape, dtype, dlayout, buffer_type, output_mem_config, dat
             ttnn.MemoryConfig(ttnn.TensorMemoryLayout.INTERLEAVED, buffer_type[2]),
             dtype[2],
         )
-        t3 = ttl.tensor.addcdiv(t0, t1, t2, scalar, output_mem_config)
+        t3 = ttnn.addcdiv(t0, t1, t2, value=scalar, memory_config=output_mem_config)
```

Comment:
Did u tried running in local? ttnn is not imported?

Reply:
no. its a non working folder. ttnn is used previously but its not imported.